### ISSUE TITLE
Adjust height for each slide

### ISSUE THREAD

**Issue by agutiernc** [blaze-slider/blaze-slider#47]
Is there a way to adjust the height for each slide depending on content?

When I target .blaze-track in a CSS file, I can change the height, but all the slides are the same height.
I also tried  targeting each slide div with different heights, but it doesn't work.

I'm trying to implement this in React. Having a height auto feature for each individual slide would help.

**Comment by erikyo** [comment]
@agutiernc the flex cs property already sets the same height for blaze-track children, I think you are referring to images placed inside the slide div. In this case you must set the image css like below>

```css
.blaze-slider .blaze-track > div img {
    height: 100%;
    -o-object-fit: cover;
    object-fit: cover;
}
```

hope it helps!

**Comment by agutiernc** [comment]
That wasn't the issue. This is my situation. I have 3  slides, each slide  should be different sizes, depending on the amount of content. Each of the slides fills up with a grid of cards (4 columns and the rows is dependent on the number of cards). The slide with the largest number of cards seems to set the size for all the other slides, even if they have a few cards.

 With Blaze Slider, it seems that all the slides are the same size. It will use the size of the largest slide to make the other slides the same size. I tried tweaking them with CSS, but nothing.

I would like for each slide to  have its own independent height.

[Embla Carousel has an AutoHeight plugin](https://www.embla-carousel.com/plugins/auto-height/) to do this. I tried it and it works. It would be great if I can do this in Blaze Slider too.
